JSDemos/Demos/PivotGrid/FieldPanel/description.md

```diff
@@ -1 +1,16 @@
-The PivotGrid allows its users to&nbsp;sort and filter data, or&nbsp;manage the collection of&nbsp;fields at&nbsp;runtime. All these operations can be&nbsp;performed using the field panel located at&nbsp;the top of&nbsp;the component. Click the sorting or&nbsp;filtering icon of&nbsp;one of&nbsp;the fields to&nbsp;sort or&nbsp;filter data by&nbsp;this field. Right-click the &laquo;Sales&raquo; field and select an&nbsp;item from the appeared context menu to&nbsp;change the summary type. Drag and drop the fields to&nbsp;rearrange them.
\ No newline at end of file
+The PivotGrid allows its users to&nbsp;sort and filter data, or&nbsp;manage the collection of&nbsp;fields at&nbsp;runtime. All these operations can be&nbsp;performed using the field panel located at&nbsp;the top of&nbsp;the component. Click the sorting or&nbsp;filtering icon of&nbsp;one of&nbsp;the fields to&nbsp;sort or&nbsp;filter data by&nbsp;this field. Right-click the &laquo;Sales&raquo; field and select an&nbsp;item from the appeared context menu to&nbsp;change the summary type. Drag and drop the fields to&nbsp;rearrange them.
+
+A field panel is a component that displays unique pivot grid fields. Field panel allows users to manage fields at runtime which makes this component similar to the [field chooser](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser).
```

Comment:
The first sentence of the first para says what field panel does. This sentence  ("A field panel is a component that displays...") also explains the essence of field panel. It would be better to have only a single sentence of such a kind, wouldn't it?

Reply:
>The first 2 sentences of the previous para say what field panel does. This sentence ("A field panel is a component that displays...") also explains the essence of field panel. It would be better to have only a single sentence of such a kind, wouldn't it?

Sorry, I left the first paragraph that should be removed.

>The doc for fieldPanel says that "The field panel is a component that displays the fields involved in the calculation of grid summaries." But here's a PivotGrid that has no summaries, but uses field panel: https://codepen.io/PavelDX/pen/abWaraL Thus the quoated sentence is not always appliable?

I'm pretty sure that the quotated sentence is just excessive. I completely agree that it is not always appliable.

>Should this be added to this demo?

Good idea. Seems like I should tell more about field chooser.